docs/kane-ai-automation-code-generation.md

```diff
@@ -0,0 +1,87 @@
+---
+id: kane-ai-automation-code-generation
+title: KaneAI Automation Code Generation
+hide_title: false
+sidebar_label: Automation Code Generation
+description: Explore KaneAI’s features and components to generate automation test code.
+keywords:
+  - lambdatest automation
+  - lambdatest kaneai
+  - kaneai code gen
+url: https://www.lambdatest.com/support/docs/kane-ai-automation-code-generation
+site_name: LambdaTest
+slug: kane-ai-automation-code-generation/
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<script type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify({
+       "@context": "https://schema.org",
+        "@type": "BreadcrumbList",
+        "itemListElement": [{
+          "@type": "ListItem",
+          "position": 1,
+          "name": "Home",
+          "item": "https://www.lambdatest.com"
+        },{
+          "@type": "ListItem",
+          "position": 2,
+          "name": "Support",
+          "item": "https://www.lambdatest.com/support/docs/"
+        },{
+          "@type": "ListItem",
+          "position": 3,
+          "name": "KaneAI Code Gen",
+          "item": "https://www.lambdatest.com/support/docs/kane-ai-automation-code-generation/"
+        }]
+      })
+    }}
+></script>
+KaneAI generates automation script is multiple frameworks and languages based on natural language inputs from users. KaneAI is available for desktop web applications as well as native mobile applications currently.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/generate-code-button.png').default} alt="generated-code-listing" className="doc_img"/>
+Click the `Generate New Code` button to go to code generation page where you can select desired framework and language.
+
+<img loading="lazy" src={require('../assets/images/kane-ai/generate-code-page.png').default} alt="code-generation-dashboard" className="doc_img"/>
+
+
+## Web Frameworks
+
+### Selenium
+Selenium is your go-to tool for cross-browser testing, enabling automated testing across different web browsers like Chrome, Firefox, Safari, and Edge. It's perfect for end-to-end testing of web applications. With Kane AI, you can generate Selenium code in:
+
+- **Python:** For Web devices this is **generated by default**.
```

Comment:
Reword this. We don't use words such as "this" in documents. 

Mention something like. By default for all web tests authored in KaneAI, an automation script on Selenium Python with PyTest framework is generated. @Manuraj-7